cmd/logger.go

```diff
@@ -137,6 +155,24 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
+func printJSON(level, source, cause, msg string, stack []string, data ...interface{}) {
+	log := &logEntry{
+		Level:   level,
+		Source:  source,
+		Stack:   stack,
+		Cause:   cause,
+		Time:    time.Now().UTC().Format(time.RFC3339Nano),
+		Message: fmt.Sprintf(msg, data...),
+	}
+
+	buf, _ := json.Marshal(log)
```

Comment:
Variable name `buf` is meaningless here. 
Call it something more meaningful in this context, like `log_json` or something.

Reply:
done